straxen/plugins/veto_hitlets.py

```diff
@@ -120,6 +122,23 @@ def compute(self, records_nv, start, end):
         return hitlets
 
 
+def remove_switched_off_channels(hits, channel_range, to_pe):
+    """Removes hits which were found in a channel without any gain.
+
+    :param hits: Hits found in records.
+    :param channel_range: Tuple of min/max channel. E.g. (2000, 2119)
+        for the neutron-veto.
+    :param to_pe: conversion factor from ADC per sample.
+    :return: Hits
+    """
+    channel_on = np.argwhere(to_pe > 0).flatten()
+    all_channel = np.arange(channel_range[0], channel_range[1] + 1)
+    channel_off = set(all_channel) - set(channel_on)
+    mask_on = ~np.isin(hits['channel'], list(channel_off))
+    hits = hits[mask_on]
+    return hits
```

Comment:
So if this is really impacting performance, you could of course make it simpler still, but ok, this is semantics
```suggestion
    channel_off = np.argwhere(to_pe[channel_range[0]: channel_range[1]] == 0).flatten() + channel_range[0]
    mask_off = np.isin(hits['channel'], channel_off)
    hits = hits[~mask_on]
    return hits
```